book/image-controller.md

```diff
@@ -1,2 +1,141 @@
 # Image Controller
 
+## Overview
+
+The Image Controller for AppStudio helps set up container image repositories
+for AppStudio Components. With image controller, AppStudio ensures one image
+repository per Component and each build pipeline will have dedicated secret to
```

Comment:
not for each build pipeline, each pipeline/pipelineRun for the compont is using the same repository.